src/Microsoft.Health.Core/Features/Security/RoleLoader.cs

```diff
@@ -62,7 +62,7 @@ public Task StartAsync(CancellationToken cancellationToken)
             };
 
             validatingReader.ValidationEventHandler += (sender, args) =>
-                throw new InvalidDefinitionException(string.Format(Resources.ErrorValidatingRoles, args.Message));
+                throw new InvalidDefinitionException(string.Format(CultureInfo.InvariantCulture, Resources.ErrorValidatingRoles, args.Message));
```

Comment:
Often our format string args are missing the explicit culture. For `string` args, the culture is irrelevant as the `string` is already a `string` and cannot be formatted for a given culture. `int` also doesn't vary much (if at all? I cannot think of anything). So I think Invariant is fine

Reply:
I don't think we localize any of our resources in this codebase now because it's all based on API calls that have no culture information passed. Because of this behavior, we should be defaulting to en-us from https://github.com/microsoft/healthcare-shared-components/blob/master/tools/SchemaManager.Core/AssemblyInfo.cs

Reply:
Ah yes! So `InvariantCulture` is appropriate then (as would something like `CultureInfo.GetCultureInfo("en-us")` or `CultureInfo.GetCultureInfo("en")` ).